src/app.module.ts

```diff
@@ -1,10 +1,26 @@
 import { Module } from '@nestjs/common';
-import { UserController } from './user/user.controller';
-import { UserService } from './user/user.service';
+import { ConfigModule } from '@nestjs/config';
+import { SequelizeModule } from '@nestjs/sequelize';
+import { User } from './user/models/user.model';
+import { UserModule } from './user/user.module';
 
 @Module({
-  imports: [],
-  controllers: [UserController],
-  providers: [UserService],
+  imports: [
+    ConfigModule.forRoot({ envFilePath: '.env' }),
+    SequelizeModule.forRoot({
+      dialect: 'postgres',
+      host: process.env.POSTGRES_HOST,
+      port: Number(process.env.POSTGRES_PORT),
+      username: process.env.POSTGRES_USERNAME,
+      password: process.env.POSTGRES_PASSWORD,
+      database: process.env.POSTGRES_DB,
+      models: [User],
+      autoLoadModels: true,
+      synchronize: true,
```

Comment:
in case you do not need timestamps in your user model, you can add here
`      define: {
        timestamps: false,
      },`

Reply:
added and removed timestamps